### PR TITLE
Pass brand parameter down to Notifications

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: src/SdkConfig.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+class SdkConfig {
+
+    static get() {
+        return global.mxReactSdkConfig;
+    }
+
+    static put(cfg) {
+        var defaultKeys = Object.keys(DEFAULTS);
+        for (var i = 0; i < defaultKeys.length; ++i) {
+            if (cfg[defaultKeys[i]] === undefined) {
```
Comment (richvdh) [inline src/SdkConfig.js]: `var key = defaultKeys[i]` would be nice here

File: src/SdkConfig.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                cfg[defaultKeys[i]] = DEFAULTS[defaultKeys[i]];
```
Comment (richvdh) [inline src/SdkConfig.js]: not entirely convinced that modifying the input object is a polite thing to do here, but <shrug>

**Comment by richvdh** [comment]
I'm afraid I don't really get this (which may be not due to seeing the context it comes from, but the lack of comments on `SdkConfig` isn't helping). Baking in the assumption that there can only ever be one MatrixChat in the VM doesn't seem like a good idea if we can possibly help it; and in any case `MatrixChat` looks like the wrong place to be storing the config - why isn't the singleton object populated from `vector/index.js` when the config is loaded?

File: src/SdkConfig.js (hunk 2 above)
Comment (dbkr) [inline src/SdkConfig.js]: Yeah, I thought about it but decided it was going to be unlikely anyone would ever be passing in something to this that shouldn't be modified.

**Comment by dbkr** [comment]
Agree it would make more sense for the config to be set somewhere other than MatrixChat since this means you can't configure the SDK if you just use lower-level components. I think this is a separate concern though as this doesn't change the interface here.

Also agree about breaking multiple MatrixChats, although we already do this with the matrix client by storing it as a singleton.

Will add some explanatory text to SdkConfig.

**Comment by richvdh** [comment]
Let the record show that there was a discussion on #matrix-core about the evilness of singletons, and whether there was any better way to achieve this end (such as passing the config object around everywhere, or using react-context).

Dave was going to experiment a bit.

**Comment by richvdh** [comment]
lgtm.

**Pull request merged**